kotlin/android/app/src/main/res/font/font.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<font-family xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+    <font
```

Comment:
Ah, we decided to go with `Source Sans Pro` for 1.0. I think a FIXME here is ok.

```suggestion
    <!-- FIXME: Use Source Sans Pro for 1.0 -->
    <font
```